modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/viewmodel/AutoDownloadSettingsViewModel.kt

```diff
@@ -24,6 +28,15 @@ class AutoDownloadSettingsViewModel @Inject constructor(
     override val coroutineContext = Dispatchers.Default
     private var isFragmentChangingConfigurations: Boolean = false
 
+    private var _hasEpisodesWithAutoDownloadEnabled: MutableStateFlow<Boolean> = MutableStateFlow(false)
+    val hasEpisodesWithAutoDownloadEnabled: StateFlow<Boolean> = _hasEpisodesWithAutoDownloadEnabled
+
+    init {
+        viewModelScope.launch(Dispatchers.IO) {
+            _hasEpisodesWithAutoDownloadEnabled.value = podcastManager.hasEpisodesWithAutoDownloadStatus(Podcast.AUTO_DOWNLOAD_NEW_EPISODES)
```

Comment:
Is there a chance that this value would be set after it's used in `AutoDownloadSettingsFragment.setupNewEpisodesToggle`?